### ISSUE TITLE
Dynamic WSDL address locations [SWS-47]

### ISSUE THREAD

**Issue by gregturn** [spring-projects/spring-ws#206]
**[Arjen Poutsma](https://jira.spring.io/secure/ViewProfile.jspa?name=arjen.poutsma)** opened **[SWS-47](https://jira.spring.io/browse/SWS-47?redirect=false)** and commented

One disadvantage of writing your WSDLs yourself is that you don't have dynamic address locations anymore. Sometimes, it is needed to transform the address given in the WSDL document to one that matches the current url of the war.

Basically, this feature would allow people to use a relative paths in their wsdl description (such as /services/myservice). When a request comes in, the full context path is prepended to this relative url (resulting in something like http://example.com:8080/context/services/myservice).

---

1 votes, 1 watchers
